src/getHumanAge.test.js

```diff
@@ -7,4 +7,40 @@ describe('getHumanAge', () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+
+  test('the age of both is equal 0', () => {
```

Comment:
can u give more informative description of tests: 'should return 0 for cat and dog when they are 14 years old' as en example. Change for other tests